reV/supply_curve/supply_curve.py

```diff
@@ -284,7 +291,8 @@ def _compute_lcot(trans_table, fcr, trans_costs=None, max_workers=None,
             feature = TC(trans_table, line_limited=line_limited,
                          **trans_costs)
             cost = []
-            for _, row in trans_table.iterrows():
+            iterrows = trans_table[~pd.isna(trans_table['sc_gid'])].iterrows()
```

Comment:
Since your already defining `mask = ~pd.isna(trans_table['sc_gid'])` below, why not have this be:
```
mask = ~pd.isna(trans_table['sc_gid'])
for _, row in  trans_table[mask].iterrows():
```

Reply:
Agreed, moved the `gid_mask` up to the top of the method